src/config.js

```diff
@@ -1,21 +1,46 @@
 /* eslint-disable import/no-dynamic-require */
 /* eslint-disable global-require */
 const path = require('path')
+const joi = require('joi')
 
-if (!process.env.NODE_ENV) { process.env.NODE_ENV = 'development' }
+const nodeEnvSchema = joi.object({
```

Comment:
so I get what is happening here but we will not need to have a validation for this.

That is because we're already checking if its undefined and setting it unlike the rest.